### PR TITLE
Replaced deprecated table attributes with css

### PR DESCRIPTION
**Pull request by mikeschinkel**
HTML deprecated table element attributes @cellpadding, @cellspacing, @border, @align, etc. so this PR replaces them with CSS classes, selectors and style rules.  It also removed a CSS style rule that was being overwritten by the next line, it removes an invalid font rule (See https://stackoverflow.com/a/20641079/102699), it removes redundant `px` from `0px` and it sneaks in a <label> element to wrap an <input> element per PhpStorm's code inspection instructions.

### CODE REVIEW

File: mailing-lists.php [hunk 1]
```diff
@@ -345,8 +345,9 @@ function output_lists_table($mailing_lists)
 <?php output_lists_table($internals_mailing_lists); ?>
 
 <p class="center">
- <strong>Email:</strong>
- <input type="text" name="email" size="40" value="user@example.com">
+ <label for="email"><strong>Email:</strong>
```
Comment (kelunik) [inline mailing-lists.php:348]: There's no need for the `for` attribute if the label wraps the input.

**Review by kelunik — APPROVED** [review]

File: license/index.php [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
 <ul>
  <li>
-  PHP 4, PHP 5 and PHP 7 are distributed under the
+  Starting with PHP 4, versions of the PHP software are distributed under the
```
Comment (mikeschinkel) [inline license/index.php:30]: This is already in master but I evidently didn't rebase to master before this PR.

File: mailing-lists.php (hunk 1 above)
Comment (mikeschinkel) [reply mailing-lists.php]: @kelunik -— `for` removed.

**Comment by derickr** [comment]
This conflicts. Can you rebase on master?

**Pull request closed without merging**